SMSService/src/lambdas/hello/hello.js

```diff
@@ -0,0 +1,60 @@
+import AWS from "aws-sdk";
+import logger from "../utils/logger.js";
+import { checkQueueMessage } from "./utils/checkQueueMessage.js";
+import { handleError } from "../utils/handleError.js";
+import { publishMessage } from "./utils/publishMessage.js";
+
+const sns = new AWS.SNS();
+
+export const hello = async (event) => {
+  try {
+    const messages = event.Records;
```

Comment:
we have a batch size of 1 and only work with 1 record. 
```suggestion
    const message = event.Records;
```